docs/resources/lke_node_pool.md

```diff
@@ -60,6 +60,10 @@ resource "linode_lke_cluster" "my-cluster" {
     label       = "my-cluster"
     k8s_version = "1.28"
     region      = "us-mia"
+
+    labels = {
+        "key" = "value"
+    }
     
```

Comment:
I think `taint` is not a very common for basic examples because it aims somehow avoiding the nodes in the pool to be scheduled a pod. You probably only want to do something like that in a larger scale environment, right?